pkg/scheduler/api/numa_info.go

```diff
@@ -253,3 +255,9 @@ func (resSets ResNumaSets) Clone() ResNumaSets {
 
 	return newSets
 }
+
+// ScoredNode is the wrapper for node during Scoring.
+type ScoredNode struct {
+	NodeName string
+	Score    int64
```

Comment:
what is the intention of adding this structure?

Reply:
Describe a node and its score. For constructing slice of scoredNode instead of map[string]score

Reply:
I think it is more readable comparing with map.